build/configs/rtl8721csm/scripts/rlx8721d_img2_ns.ld

```diff
@@ -383,13 +383,15 @@ SECTIONS
 	.psram_image2.data :
 	{
 		. = ALIGN (32);
+		__psram_data_start__ = .;
```

Comment:
This is not our naming policy. Could you check Thapa's previous change?

Reply:
Hi Chang,
I had added only bss start and end variables and internal & extrenal heap start variables.
https://github.com/Samsung/TizenRT/blob/master/docs/HowToAddnewBoard.md#configs